src/System.Runtime.Numerics/tests/BigInteger/cast_from.cs

```diff
@@ -479,7 +479,7 @@ public static void RunSingleExplicitCastFromBigIntegerTests()
             VerifySingleExplicitCastFromBigInteger(Single.MaxValue, bigInteger);
 
             // Single Explicit Cast from BigInteger: Random value > Single.MaxValue
-            bigInteger = GenerateRandomBigIntegerGreaterThan(Single.MaxValue * 2, s_random);
+            bigInteger = GenerateRandomBigIntegerGreaterThan((double)Single.MaxValue * 2, s_random);
```

Comment:
This test was failing due to a difference in how the new compiler was generating IL.
This is what was being generated by the roslyn compiler:
```c#
bigInteger = cast_fromTest.GenerateRandomBigIntegerGreaterThan(double.PositiveInfinity, cast_fromTest.s_random);
```
This was fixed by casting it to double.